lib/v2/jasa/section.js

```diff
@@ -0,0 +1,105 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const path = require('path');
+const { art } = require('@/utils/render');
+
+module.exports = async (ctx) => {
+    const host = 'https://asa.scitation.org/toc/jas/';
+    const hostUrl = host.concat('current?size=all');
+    const res = await got('https://asa.scitation.org');
+    const cookie = res.headers['set-cookie'].join(' ');
+
+    const currResponse = await got({
+        method: 'get',
+        url: hostUrl,
+        headers: {
+            'Accept-Encoding': 'gzip, deflate, br',
+            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.81 Safari/537.36',
+            cookie,
+        },
+    });
+    const $1 = cheerio.load(currResponse.data);
+    const baseUrl = $1('.row.js_issue.highlighted').find('a').attr('href');
+    const heading = $1('.subject-heading')
+        .map((_, item) => ({
+            section: $1(item).text(),
+        }))
+        .get();
+
+    let list;
+    let ifContainSection = false;
+    const id = ctx.params.id.split('+').join(' ');
+    for (let i = 0, l = heading.length; i < l; i++) {
+        if (heading[i].section === id) {
+            ifContainSection = true;
+        }
+    }
+    if (ifContainSection === true) {
+        const sectUrl = baseUrl.concat('?tocSection=', id);
+
+        const response = await got({
+            method: 'get',
+            url: sectUrl,
+            headers: {
+                'Accept-Encoding': 'gzip, deflate, br',
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.81 Safari/537.36',
```

Comment:
https://github.com/DIYgod/RSSHub/blob/a43b67bc1f6d04519c09ac04f3fce4c9e074381a/lib/config.js#L40

`UA` should be configured via env instead of hard-coded.

Reply:
Thanks your attention, I have modified it.